modules/gapi/include/opencv2/gapi/infer/ie.hpp

```diff
@@ -351,6 +362,29 @@ template<typename Net> class Params {
         return *this;
     }
 
+    /** @brief Specifies the output precision for model.
+
+    The function is used to set an output precision for model.
+
+    @param precision Precision in OpenCV format.
+    @return reference to this parameter structure.
+    */
+    Params<Net>& cfgOutputPrecision(detail::ParamDesc::precision_t precision) {
```

Comment:
ah this mixture of CamelCase and snake_case_t :)

Reply:
No problem

Reply:
Done